### DISCUSSION TITLE
What midi keyboards are supported

### DISCUSSION THREAD

**Discussion by wilbert-alberts** [probonopd/MiniDexed#201] [Q&A] [answered]
Hi,

Thanks for this fantasic project! 

Question: I tried to use my Yamaha reface DX as 'midi' keyboard. (I used it in the context of the Ardour DAW where it works flawlessly.) I couldn't get it to work in minidexed. Using another (but worse) keyboard did work. 

Is there a limitation on the keyboards that can be used (being connected via USB)? How can I check whether a particular midi is supported?

Regards,
Wilbert.

**Comment by probonopd** [comment, marked as answer]
Hello @wilbert-alberts 
Using a Yamaha reface DX as a keyboard for MiniDexed, wow! :+1: 

Assuming that the Yamaha reface DX is a _class compliant_ USB MIDI controller (meaning: it needs no extra drivers on the PC) it _should_ work.

Did you try to remove `usbspeed=full` from `cmdline.txt`?

**Reply by wilbert-alberts** [reply]
Hi,
Yep, removing that line did the trick. Now I can enjoy the sound of the 'real DX7' with a master keyboard that tries to emulate one. Both sound very nice.
Thanks a lot!
Regards,
Wilbert
